src/main/java/algorithms/curated170/hard/PaintHouseII.java

```diff
@@ -0,0 +1,36 @@
+package algorithms.curated170.hard;
+
+public class PaintHouseII {
```

Comment:
This also came along.

Reply:
Sorry for that. It will not affect but disturbing.